src/Ens/JobeetBundle/Entity/Affiliate.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace Ens\JobeetBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Affiliate
+ */
+class Affiliate
+{
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var string
+     */
+    private $url;
+
+    /**
+     * @var string
+     */
+    private $email;
+
+    /**
+     * @var string
+     */
+    private $token;
+
+    /**
+     * @var \DateTime
+     */
+    private $created_at;
+
+    /**
+     * @var \Doctrine\Common\Collections\Collection
+     */
+    private $category_affiliates;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->category_affiliates = new \Doctrine\Common\Collections\ArrayCollection();
```

Comment:
@PhotonSapiens 
using fully qualified namespaces makes code non-readable - it should be 
```PHP
use Very\Long\And\Boring\Namespace\Nobody\Cares\About\Where\Is\MyClass

new MyClass();

```